dashboard/config/foorm/forms/surveys/pd/summer_workshop_post_survey.0.json

```diff
@@ -459,6 +897,25 @@
      "allowAddPanel": false,
      "allowRemovePanel": false
     },
+    {
+     "type": "html",
+     "name": "clickTo119",
+     "html": "<p><span style=\"font-size: 14pt; font-family: verdana, geneva, sans-serif;\"><strong>Overall Workshop Feedback</strong></span></p>\n<p>We’d like to hear your feedback about your experience this week overall, <Strong>especially how this professional development could better fit your learning needs, or how it could be improved</strong>.\n"
+    },
+    {
+     "type": "comment",
+     "name": "activities_you_liked",
+     "title": "What were the one or two things you liked most about the activities you did in this workshop and why?",
+     "cols": 6,
+     "rows": 6
+    },
+    {
+     "type": "comment",
+     "name": "activities_youd_change",
+     "title": "What are the one or two things you would change about the activities you did in this workshop?\t",
+     "cols": 60,
+     "rows": 6
+    },
     {
      "type": "radiogroup",
      "name": "iGive",
```

Comment:
update options for this question to be value/text objects

Reply:
fixed